src/main/webapp/js/common.js

```diff
@@ -558,17 +558,13 @@ function scrollToTop(duration) {
 var DIV_STATUS_MESSAGE = '#statusMessagesToUser';
 
 /**
- * Sets a status message and the message status.
+ * Prepares a status message and the message status.
  * Default message type is info.
  *
  * @param message the text message to be shown to the user
  * @param status type
  */
-function setStatusMessage(message, status) {
-    if (message === '' || message === undefined || message === null) {
-        return;
-    }
-
+function prepareStatusMessage(message, status) {
     var $statusMessagesToUser = $(DIV_STATUS_MESSAGE);
```

Comment:
Since we are touching this code, perhaps rename this to `$statusMessageDiv` for clarify